src/js/components/ComboBox/Combobox.tsx

```diff
@@ -58,6 +59,23 @@ interface Filter {
 
 type Props = PropTypes.InferProps<typeof propTypes>;
 
+const TooltipWrapper = ({ tooltipLink, children }) => {
+  if (!tooltipLink) {
+    return <>{children}</>;
+  }
+
+  return (
+    <a href={tooltipLink} target="_blank" rel="noreferrer">
```

Comment:
was looking up noreferrer, do you think noopener would be beneficial here too? https://www.reliablesoft.net/noreferrer-noopener/#noopener